tests/gen/long_time_style_custom_non_recent_none_nix.stderr

```diff
@@ -1,2 +1,2 @@
-thread 'main' panicked at 'Custom timestamp format is empty, please supply a chrono format string after the plus sign.', src/options/view.rs:345:47
```

Comment:
these checks are kinda stupid. the tests shouldn't assume a particular line of error. not sure if it would be easy to make them more robust though.

Reply:
These tests are getting more robust in #787 which is a whole refactor of the args parsing